src/libraries/Microsoft.PowerFx.Core/Public/Config/ReadOnlySymbolTable.cs

```diff
@@ -402,6 +402,24 @@ IEnumerable<TexlFunction> INameResolver.LookupFunctionsInNamespace(DPath nameSpa
             return _functions.WithNamespace(nameSpace);
         }
 
+        internal virtual void EnumerateNames(List<SymbolEntry> names, EnumerateNamesOptions opts)

```

Comment:
>List [](http://example.com/codeflow?start=45&length=4)

IList<>?

Reply:
IList doesn't have Add() !

Reply:
I'm wrong - it is present, via ICollection<T>